pyslurm/slurm.pxd

```diff
@@ -1,42 +1,50 @@
-# cython: embedsignature=True
```

Comment:
Don't we want to keep this directive?

Reply:
I am not sure, what that directive does, I created the slurm.pxd or better the slurm.j2 from scratch and didn't need it.

Reply:
I checked the BUILD.md again and from my point of view the slurm.pxd is no longer necessary, because it would build from the jinja2 template slurm.j2.

Reply:
This directive is used on the Python REPL. Do a dir() with and without and you'll see the difference.